manim/mobject/svg/text_mobject.py

```diff
@@ -676,8 +676,45 @@ class MarkupText(SVGMobject):
     r"""Display (non-LaTeX) text rendered using `Pango <https://pango.gnome.org/>`_.
 
     Text objects behave like a :class:`.VGroup`-like iterable of all characters
-    in the given text. In particular, slicing is possible. Text can be formatted
-    using different tags:
+    in the given text. In particular, slicing is possible.
```

Comment:
Like in `text_mobject.py`, you should probably add a reference to the tutorial here too.